packages/python/src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { SourceFileContext } from "@alloy-js/core";
+import { PythonElements, usePythonNamePolicy } from "./name-policy.js";
+
+export function getFormattedName(
```

Comment:
I'm not against this function, but generally having a utils.ts. This function is a single line, so not sure we're doing any DRY-ing by replacing the underlying logic.

Reply:
Removed.